utils/src/treev2/quickPickWizard/quickPickAzureResource/tempTypes.ts

```diff
@@ -3,22 +3,22 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { ApplicationResource, ApplicationSubscription, ResourceGroupsItem } from '../../../../hostapi.v2';
+import { ApplicationResource, ApplicationSubscription } from '../../../../hostapi.v2';
 import * as types from '../../../../index';
 
 // TODO: THIS FILE IS TEMPORARY //
 // It needs to be replaced by real Resources extension interfaces //
 
 // These are assumptions made about the nodes in the tree
 
-export type SubscriptionItem = ResourceGroupsItem & {
+export type SubscriptionItem = {

```

Comment:
These should probably all stay, and also `ResourceGroupsItem` should be in this file for now.

Reply:
> Approved but _maybe_ the other subtypes should keep `ResourceGroupsItem` (in the comment).

Ok, added it back so we have 

```ts
export type SubscriptionItem = ResourceGroupsItem & {
    subscription: ApplicationSubscription;
}

export type GroupingItem = ResourceGroupsItem & {
    resourceType?: types.AzExtResourceType
}

export type AppResourceItem = ResourceGroupsItem & {
    resource: ApplicationResource;
};

export type ResourceGroupsItem = unknown;
```